sources/packages/backend/apps/api/src/route-controllers/overaward/_tests_/e2e/overaward.institution.controller.getOverawardBalance.e2e-spec.ts

```diff
@@ -0,0 +1,67 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import {
+  createFakeStudent,
+  createFakeDisbursementOveraward,
+} from "@sims/test-utils";
+import { Repository } from "typeorm";
+import {
+  DisbursementOveraward,
+  DisbursementOverawardOriginType,
+  Student,
+} from "@sims/sims-db";
+
+import {
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getInstitutionToken,
+  InstitutionTokenTypes,
+} from "../../../../testHelpers";
+
+describe("OverawardInstitutionController(e2e)-getOverawardBalance", () => {
+  let app: INestApplication;
+  let studentRepo: Repository<Student>;
+  let disbursementOverawardRepo: Repository<DisbursementOveraward>;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    studentRepo = dataSource.getRepository(Student);
+    disbursementOverawardRepo = dataSource.getRepository(DisbursementOveraward);
+  });
+
+  it("Should return correct value for overaward balance", async () => {
+    // Arrange
+    const student = await studentRepo.save(createFakeStudent());
```

Comment:
I see that the studentRepo is used only to create/persist the student at once, would recommend to use 

```ts
const student = await saveFakeStudent(appDataSource);
```